draft-ietf-masque-h3-datagram.md

```diff
@@ -333,13 +333,14 @@ drop that Capsule.
 
 ### The REGISTER_DATAGRAM_CONTEXT Capsule {#register-capsule}
 
-The REGISTER_DATAGRAM_CONTEXT capsule (see {{iana-types}} for the value of the
-capsule type) allows an endpoint to inform its peer of the encoding and
-semantics of datagrams associated with a given context ID. Its Capsule Data
-field consists of:
+The REGISTER_DATAGRAM_CONTEXT capsule (type=0xff37a1) allows an endpoint to
+inform its peer of the encoding and semantics of datagrams associated with a
+given context ID. Its Capsule Data field consists of:
```

Comment:
`Its Capsule Data field consists of` is no longer correct, we should rephrase it

Reply:
I vote for just remove it

Reply:
Fine by me